src/utils/Options.ts

```diff
@@ -1,10 +1,10 @@
 module n3Charts.Utils {
   'use strict';
 
-  export interface IAxes {
-    x: AxisOptions;
-    y: AxisOptions;
-    y2?: AxisOptions;
+  export interface IAxesSet {
```

Comment:
No need for the word "set" here, in my opinion : "axes" is the plural form of "axis" anyway.